Sources/Paywalls/Components/PaywallComponentBases.swift

```diff
@@ -98,16 +79,6 @@ public enum PaywallComponent: Decodable, Sendable, Hashable, Equatable {
             return component.displayPreferences
         case .image(let component):
             return component.displayPreferences
-        case .video(let component):
```

Comment:
This whole `displayPreferences` could honestly probably get deleted. This was for the portrait/landscape POC that we will probably end up rethinking Its just a bit messy ATM so I'm find getting rid of this 👍